raiden/tests/scenarios/bf2_long_running.yaml

```diff
@@ -0,0 +1,327 @@
+version: 2
+
+settings:
+  gas_price: "fast"
+  chain: goerli
+  services:
+    pfs:
+      url: https://pfs-goerli.services-dev.raiden.network
+    udc:
+      enable: true
+      token:
+        deposit: true
+
+token:
+  address: "0x62083c80353Df771426D209eF578619EE68D5C7A"
+  balance_fund: 200_000_000_000_000_000_000
```

Comment:
200 ETH is probably quite a lot more than you need. You could just have copy pasted the values from the other scenarios. I don't think it's very realistic to have people depositing 100 ETH into a payment channel.